README.md

```diff
@@ -26,6 +29,27 @@ will be just about your specific repository.
 - `metrics_files`: A list of relative paths to `metrics.yaml` files
 - `ping_files`: A list of relative paths to `pings.yaml` files
 
+There is also an `apps` list to define the concrete applications in the family.
+In most cases, there will be only one entry under `apps`, but more complex situations
+like Fenix have separate apps for different logical release channels.
+Each application can override specific fields like `description` from the family, but
```

Comment:
only some or all can be overwritten?
I assume e.g. `url` can't. This should be clarified here.